src/assets/donors/index.js

```diff
@@ -0,0 +1,33 @@
+import EthereumLogo from './Ethereum.svg'
+import ECFLogo from './ECF.svg'
+import district0xLogo from './district0x.svg'
+import SpankChainLogo from './SpankChain.svg'
+
+const donors = [
+  {
+    logo: EthereumLogo,
+    name: "Ethereum Foundation",
+    website: "https://www.ethereum.org/foundation",
+    twitter: "https://twitter.com/ethereum"
```

Comment:
Is this property used somewhere?

Reply:
I was considering it early on but didn’t end up doing it. Will remove